SPECS/oniguruma/oniguruma.spec

```diff
@@ -1,15 +1,14 @@
+%define subversion 1
```

Comment:
I know it's still a draft but since I already opened this PR: I think there's no need to treat that `1` as anything special - it looks like a regular part of the version. While it would be nice, if all projects worldwide followed the same schema, we already deal with a whole bunch of different ways to version a project.:) Unless there's a need to use that `subversion` value for a special purpose (example: build command argument) but it doesn't look like it at the moment.

Reply:
it's only used in the source url. Should i hardcode the "1" in the url?

Reply:
Within the onigurama.6.9.7.1.tar.gz, the root folder is onig-6.9.7. The build fails with a directory not found error if i include the subversion in the version number. fedora seems to also treat the subversion differently for the same reason i believe. https://src.fedoraproject.org/rpms/oniguruma/blob/rawhide/f/oniguruma.spec

Reply:
Would there be a better way to approach this issue?

Reply:
I'd recommend using `cut` to take the first three components out of the version string. Try:

```
%define majmin %(echo %{version} | cut -d. -f1-3)
```

Reply:
(With a more sensible macro name than `majmin`, though)